packages/datepickers/src/styled/StyledDatepicker.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled from 'styled-components';
+import { retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+const COMPONENT_ID = 'datepickers.datepicker';
+
+const StyledDatepicker = styled.div.attrs({
+  'data-garden-id': COMPONENT_ID
+})<{
+  isSmall: boolean;
+}>`
+  background-color: ${props => props.theme.colors.background};
+  padding: ${props =>
+    `${props.isSmall ? props.theme.space.base * 4 : props.theme.space.base * 5}px`};
+  color: ${props => props.theme.colors.foreground};
+
+  direction: ${props => props.theme.rtl && 'rtl'};
```

Comment:
I've removed the space, but stylelint is saying that this ordering is correct.

Reply:
The value for the `direction` attribute can be either `false` or `"rtl"`, which are two different types. It probably doesn't affect the UI, but should this line be re-written as:

```tsx
  direction: ${props => props.theme.rtl ? 'rtl': 'ltr'};
```

I'm curious why TypeScript didn't complain here, I'm guessing it's because the value for `direction` is a function that gets evaluated? Or maybe I'm misunderstanding how this works. TypeScript does complain about using the `&&` in CSS over [here](https://github.com/zendeskgarden/react-containers/pull/111#discussion_r332165935) in the react-containers repo.

Reply:
`styled-components` has some 🌈 "magic" which helps decide when to apply your style rule.

In this case `false` (or `undefined` as well) are not valid values for the `direction` style rule and are not applied. This stops you from accidentally having `color: undefined`'s everywhere.

Reply:
Great thanks for the clarification 👍 .